### ISSUE TITLE
Functional tests should not import static data

### ISSUE THREAD

**Issue by oliverklee** [Nimut/TYPO3-testing-framework#106]
Importing the static data of extensions (e.g., static_info_tables) can be really slow, and hence slowing down the functional tests immensely. I propose the TF either does not import static data at all, or to make this configurable.

**Comment by IchHabRecht** [comment]
Hi @oliverklee,

I don't think this is a good option. If your tests depend on an extension with static data, of course you need those to get a fully working dependency. The testing framework does not know which part of the defined extensions your tests need, thus it should always enforce fully working extension setup.

**Comment by oliverklee** [comment]
@IchHabRecht Fair point. My main issue is with the performance penalty of having static tables in functional tests: Currently, having the new oelib static tables slows down the tests from about 2 seconds each to about 20 seconds each. Maybe there is some other way to remove/reduce this performance penalty?

**Comment by oliverklee** [comment]
I'll have a look at what exactly makes the tests so slow on my setup (and why this doesn't happen consistently).

**Comment by lolli42** [comment]
Note: https://github.com/TYPO3/testing-framework/ solved this.
